### PR TITLE
Update jsxgraph info

### PR DESCRIPTION
**Pull request by alfredwassermann**
# Pull request template for new library

Pull request for issue: release of v1.00.0

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/jsxgraph/jsxgraph
* Official website (optional, not the repository): https://jsxgraph.org
* NPM package url (optional): https://www.npmjs.com/package/jsxgraph
* License and its reference: MIT and LGPL 3
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 56
  * Count of stars: 507
  * Count of forks: 115
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week: 225
  * Downloads in the last month:

## Essential checklist

* [X] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [ ] This lib was not found on cdnjs repo
* [ ] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [X] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [X] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [X] The parent of the commit(s) in the PR is not older than 3 days.
* [X] Pull request is sent from a non-master branch with a meaningful name.
* [X] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [X] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by alfredwassermann** [comment]
OK; I re-activated autoupdate. Removing version 1.35 is essential for us, since it comes from an ill-named tag.

**Comment by MattIPv4** [comment]
@xtuc sorry for the late mention -- any ideas on why the git auto-update bot hasn't picked this release up?

The auto-update config already present in this library seems valid to me: https://github.com/jsxgraph/jsxgraph/tree/v1.00.0/update_cdnjs

**Comment by alfredwassermann** [comment]
The 1.35 tag is meanwhile removed.

**Comment by MattIPv4** [comment]
@alfredwassermann we will never, ever allow the removal of assets that are already published on the CDN, sorry. Users may be using these assets and removing them would break trust in cdnjs.

As mentioned above as well, we are no longer accepting pull requests that touch versioned files. The only file a PR should ever touch now is a package.json file for a library.

**Comment by alfredwassermann** [comment]
So, what can I do?

**Comment by MattIPv4** [comment]
I would ask that you revert changes to versioned library files in this PR, so that you are only modifying the package.json file.

We'll need to wait for Sven to see if he can figure out why your new version hasn't been picked up.

**Comment by alfredwassermann** [comment]
@MattIPv4  ok, the folder of the 1.35 release is restored.

**Comment by MattIPv4** [comment]
@alfredwassermann you'll also need to revert the change that adds the new version as well, only package.json should be in the PR diff. Thank you!

**Comment by alfredwassermann** [comment]
@MattIPv4  Done. The 1.0.0 folder is removed.

File: ajax/libs/jsxgraph/package.json [hunk 1]
```diff
@@ -1,14 +1,24 @@
 {
   "name": "jsxgraph",
   "filename": "jsxgraphcore.js",
-  "version": "0.99.7",
```
Comment (MattIPv4) [inline ajax/libs/jsxgraph/package.json:4]: Please revert the removal of the version attr.

**Comment by alfredwassermann** [comment]
OK, done.

**Comment by MattIPv4** [comment]
Thanks for keeping cdnjs up-to-date! As for the missing version, we're currently working on a new auto-update system. Once this is live, new versions being added should be more reliable.

**Comment by alfredwassermann** [comment]
Thanks a lot for helping me out and in general for the phantastic work with cdnjs.
Does this mean, v1.00.0 will be contained in cdnjs in a few hours?

**Comment by MattIPv4** [comment]
Hey! I'm not sure when we'll pull in v1.00.0, either the existing old bot will pull it in, or we may need to wait for the new bot to add it.

> As for the missing version, we're currently working on a new auto-update system. Once this is live, new versions being added should be more reliable.

**Comment by alfredwassermann** [comment]
Sorry, but I've come back to this issue again. JSXGraph v1.00.0 is still not on cdnjs.
Meanwhile, I tested two autoupdate bots:

* `auto-update.js` from https://github.com/cdnjs/cdnjs fails to update JSXGraph. It seems to ignore github and just updates npm packages. (JSXGraph autoupdate is set for github)
* `autoupdate.js` from https://github.com/cdnjs/autoupdate/ works like a charm for JSXGraph.

**Pull request merged**